lib/stages/testDataGen.cpp

```diff
@@ -68,6 +69,10 @@ testDataGen::testDataGen(Config& config, const std::string& unique_name,
     using namespace std::placeholders;
     restServer::instance().register_post_callback(
         endpoint, std::bind(&testDataGen::rest_callback, this, _1, _2));
+
+    // Create stat tracker
+    kotekan::KotekanTrackers& KT = kotekan::KotekanTrackers::instance();
+    timer = KT.add_tracker(unique_name, "test_data_gen_timer", "sec");
```

Comment:
```suggestion
    timer = KT.add_tracker(unique_name, "frame_fill_time", "sec");
```
The unique_name should provide context for what this stage is doing, so you just need to name what metric is tracking.  In this case how long it takes to fill a frame.